addons/calendar/__openerp__.py

```diff
@@ -20,21 +19,14 @@
     """,
     'category': 'Hidden/Dependency',
     'website': 'https://www.odoo.com/page/crm',
-    'demo': ['calendar_demo.xml'],
+    'demo': ['data/calendar_demo.xml'],
     'data': [
-        'calendar_cron.xml',
+        'data/calendar_cron.xml',
+        'data/calendar_data.xml',
         'security/ir.model.access.csv',
         'security/calendar_security.xml',
-        'calendar_view.xml',
-        'calendar_data.xml',
         'views/calendar.xml',
+        'views/calendar_view.xml'
     ],
-    'qweb': ['static/src/xml/*.xml'],
```

Comment:
qweb removed then where you import qweb template, currently its throwing errors of template not found